src/scripts/main.js

```diff
@@ -1,3 +1,320 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const tableRows = tableBody.rows;
+let toggle = true;
+let valueForSort = 0;
+let special;
```

Comment:
The function 'setSalaryValue' can be simplified using an arrow function: `const setSalaryValue = num => `$${num.toLocaleString('en-US')}`;`